pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
   "pandas",
   "tiktoken>=0.8.0",
   "tabulate",
+  "mlflow",
```

Comment:
Can we do mlflow-skinny? That is the one we switched to for databricks-agents as well.